test/server/server_stats_flush_benchmark_test.cc

```diff
@@ -0,0 +1,74 @@
+#include <cstdint>
+#include <memory>
+
+#include "envoy/stats/sink.h"
+#include "envoy/stats/stats.h"
+
+#include "source/common/stats/thread_local_store.h"
+#include "source/server/server.h"
+
+#include "test/benchmark/main.h"
+#include "test/mocks/stats/mocks.h"
+#include "test/test_common/simulated_time_system.h"
+#include "test/test_common/utility.h"
+
+#include "absl/strings/str_cat.h"
+#include "benchmark/benchmark.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace Envoy {
+class StatsSinkFlushSpeedTest {
+
```

Comment:
nit: remove this blank line and put it between the `class` and `namespace` lines.

Reply:
now you added the blank line below `namespace` but there's still a superfluous one below `class`.